dftool

```diff
@@ -39,38 +44,58 @@ CHAINID_EXAMPLES = (
     f"{DEV_CHAINID} for development, 1 for (eth) mainnet, 137 for polygon"
 )
 
+# ve constants
+WEEK = 7 * 86400
+MAXTIME = 4 * 365 * 86400  # 4 years
+
 # ========================================================================
 HELP_MAIN = """Data Farming tool, for use by OPF.
 
-Usage: dftool getrate|volsym|.. ARG1 ARG2 ..
+Usage: dftool getrate|query|calc|dispense|..
```

Comment:
This change is incorrect. Suggest to revert. Eg. We _should_ have volsym, and _shouldn't_ have query.

Reply:
I see what you are referring to.